PeerioTechnologies/peerio-desktop
# fix: render fingerprint only if ContactProfile is visible

This is a quick fix for calculating fingerprint for every new contact
when opening chats.

The drawback of doing it like this is that you can see fingerprint
disappearing for a few milliseconds when the dialog is closed until
animation ends. This should be fixed in the future with larger
refactoring of how dialogs work.

#### Relevant info and issue/PR links 
 
#### Testing instructions  

Click on avatar and see if contact profile dialog shows and displays fingerprint (Peerio ID#).

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
